src/System.Net.Utilities/src/System/Net/NetworkInformation/Ping.Unix.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Threading;
+
+namespace System.Net.NetworkInformation
+{
+    partial class Ping
+    {
+        private void InternalSendAsync(IPAddress address, byte[] buffer, int timeout, PingOptions options)
+        {
+            AsyncOperation asyncOp = _asyncOp;
+            SendOrPostCallback callback = _onPingCompletedDelegate;
+
+            // TODO: Implement this
+
+            PingException pe = new PingException(SR.net_ping, new PlatformNotSupportedException());
+            var ea = new PingCompletedEventArgs(
+                new PingReply(address, default(PingOptions), default(IPStatus), default(long), buffer), 
+                pe, 
+                false, 
+                asyncOp.UserSuppliedState);
+
+            Finish();
+            asyncOp.PostOperationCompleted(callback, ea);
+        }
+
+        private void InternalDisposeCore()
+        {
+            // TODO: Implement this
+        }
+
+        // -----------------------------
```

Comment:
I'd like to keep it, at least for now.  When using this partial class scheme, it helps to separate the things that are expected by the main partial implementation from stuff which isn't directly relied upon by it.  We use the same commenting scheme in other places, and I've found it useful when I go back and make later changes.  We can always scrub them from here and other files later if they get stale or become problematic in some way.